packages/x-data-grid/src/components/cell/GridCell.tsx

```diff
@@ -373,6 +388,16 @@ const GridCell = React.forwardRef<HTMLDivElement, GridCellProps>(function GridCe
     }
   }, [hasFocus, cellMode, apiRef]);
 
+  if (isCellRowSpanned) {
+    return (
+      <div
+        data-colindex={colIndex}
+        role="presentation"
```

Comment:
This shouldn't be used, no?

```suggestion
```

Looking https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role, I see no purpose for the role here.

Reply:
Actually, this seems needed. If I understand https://www.w3.org/WAI/ARIA/apg/practices/hiding-semantics/ correctly, because the parent is a `role="row"`, it needs each child to have a cell role, since this one doesn't have it, it should be hidden.